Installation/include/CGAL/license/Weights.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2016  GeometryFactory SARL (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s) : Andreas Fabri
+//
+// Warning: this file is generated, see include/CGAL/licence/README.md
+
+#ifndef CGAL_LICENSE_WEIGHTS_H
+#define CGAL_LICENSE_WEIGHTS_H
+
+#include <CGAL/config.h>
+#include <CGAL/license.h>
+
+#ifdef CGAL_WEIGHTS_COMMERCIAL_LICENSE
+
+#  if CGAL_WEIGHTS_COMMERCIAL_LICENSE < CGAL_RELEASE_DATE
+
+#    if defined(CGAL_LICENSE_WARNING)
+
+       CGAL_pragma_warning("Your commercial license for CGAL does not cover "
+                           "this release of the  package.")
```

Comment:
Should not there be the description of the package, between `release of the` and `package`?

Reply:
You forgot to put a new entry in `Installation/include/CGAL/license/gpl_package_list.txt`. Among all files `Installation/include/CGAL/license/*.h`, only three have that sentence `this release of the  pacckage':
```
Installation/include/CGAL/license/Classification.h
Installation/include/CGAL/license/Orthtree.h
Installation/include/CGAL/license/Three.h
```

The other `.h` files have a package name in it, like:
```
this release of the Triangulated Surface Mesh Parameterization package
```

Reply:
Ooh, I see. I will do it.

Reply:
In fact, I did not forget to add the package name to the list in the file `gpl_package_list.txt`. The problem is the bug in the script `generate_files.cmake`. I created a draft PR #5778 with the fix. That also explains the same issue for the files `Classification`, `Orthtree`, and `Three`.

Reply:
It is now fixed in PR #5783.